eslintrc.json

```diff
@@ -15,8 +15,6 @@
 		"no-script-url": 2,
 		"no-shadow": 2,
 		"no-tabs": 0,
-		"no-template-curly-in-string": 0,
```

Comment:
are we supporting ES6 everywhere now or will we just override these rules in the Apps repo?